[2025-02-18T01:00:14.761+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T01:00:14.784+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: default_args_dag.task3 scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-18T01:00:14.794+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: default_args_dag.task3 scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-18T01:00:14.794+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T01:00:14.808+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task3> on 2025-02-18 00:00:00+00:00
[2025-02-18T01:00:14.814+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=10576) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-18T01:00:14.816+0000] {standard_task_runner.py:72} INFO - Started process 10577 to run task
[2025-02-18T01:00:14.817+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'default_args_dag', 'task3', 'scheduled__2025-02-18T00:00:00+00:00', '--job-id', '992', '--raw', '--subdir', 'DAGS_FOLDER/12_default_args_dag.py', '--cfg-path', '/tmp/tmpd3j9_mk9']
[2025-02-18T01:00:14.818+0000] {standard_task_runner.py:105} INFO - Job 992: Subtask task3
[2025-02-18T01:00:14.871+0000] {task_command.py:467} INFO - Running <TaskInstance: default_args_dag.task3 scheduled__2025-02-18T00:00:00+00:00 [running]> on host ab74d4d90d32
[2025-02-18T01:00:14.960+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='default_args_dag' AIRFLOW_CTX_TASK_ID='task3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T00:00:00+00:00'
[2025-02-18T01:00:14.960+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T01:00:14.974+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-18T01:00:14.975+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-18T01:00:14.984+0000] {subprocess.py:99} INFO - Output:
[2025-02-18T01:00:19.988+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-18T01:00:20.013+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T01:00:20.014+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=default_args_dag, task_id=task3, run_id=scheduled__2025-02-18T00:00:00+00:00, execution_date=20250218T000000, start_date=20250218T010014, end_date=20250218T010020
[2025-02-18T01:00:20.047+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T01:00:20.061+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T01:00:20.063+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
